Library/Formula/sip.rb

```diff
@@ -30,12 +30,15 @@ def install
 
     Language::Python.each_python(build) do |python, version|
       # Note the binary `sip` is the same for python 2.x and 3.x
-      system python, "configure.py",
-                     "--deployment-target=#{MacOS.version}",
```

Comment:
That's what I tried. It still leads to an empty space, and `configure.py` recognises it as an extra option.